hokusai/commands/check.py

```diff
@@ -0,0 +1,140 @@
+import os
+import urllib
+import shutil
+import getpass
+from distutils.dir_util import mkpath
+
+from subprocess import check_output, CalledProcessError, STDOUT
+
+from hokusai.common import print_red, print_green
+
+def check(interactive):
```

Comment:
huge function - as this is an interactive cli process, i'm not sure how easy it is to break out helpers or more sub functions but worth thinking about. e.g. you have AWS, kubectl and docker installation steps below, could be broken out potentially.